### ISSUE TITLE
Install error

### ISSUE THREAD

**Issue by rmassidda** [musikinformatik/SuperDirt#106]
I can't install SuperDirt in SuperCollider 3.10.0 (Fedora Linux 29).
Vowel and Dirt-Samples are correctly installed.
Evaluating `include("SuperDirt");` throws `ERROR: Quarks-install: path does not exist SuperDirt`.
Selecting SuperDirt in `Quarks.gui` prints this error message:
```
Installing SuperDirt
ERROR: Primitive '_ArrayAdd' failed.
Wrong type.
RECEIVER:
Instance of String {    (0xadc0f38, gc=BC, fmt=07, flg=00, set=02)
  indexed slots [5]
      0 : V
      1 : o
      2 : w
      3 : e
      4 : l
}
CALL STACK:
	MethodError:reportError
		arg this = <instance of PrimitiveFailedError>
	< closed FunctionDef >
		arg error = <instance of PrimitiveFailedError>
	Integer:forBy
		arg this = 0
		arg endval = 0
		arg stepval = 2
		arg function = <instance of Function>
		var i = 0
		var j = 0
	SequenceableCollection:pairsDo
		arg this = [*2]
		arg function = <instance of Function>
	Scheduler:seconds_
		arg this = <instance of Scheduler>
		arg newSeconds = 675.200682813
	Meta_AppClock:tick
		arg this = <instance of Meta_AppClock>
		var saveClock = <instance of Meta_SystemClock>
	Process:tick
		arg this = <instance of Main>
^^ The preceding error dump is for ERROR: Primitive '_ArrayAdd' failed.
Wrong type.
RECEIVER: Vowel
```

**Comment by santolucito** [comment]
I'm getting the same error on 3.10.0 (Ubuntu 16.04 Xenial)

**Comment by yaxu** [comment]
How have you installed SuperCollider?

**Comment by santolucito** [comment]
Yes, this is the error I get from within SuperCollider version 3.10.0 when I try to install the SuperDirt quark. SuperCollider seems to be working otherwise (testing with SinOsc.ar)

**Comment by yaxu** [comment]
I mean, how did you install SuperCollider? e.g. from system packages / compiled from source

**Comment by santolucito** [comment]
Ah sorry! That makes more sense. Installed from source

**Comment by yaxu** [comment]
I haven't been able to reproduce this problem under linux mint 

I compiled 3.10.0 using this script:
  https://github.com/lvm/build-supercollider

**Comment by yaxu** [comment]
@rmassidda Did you manage to get this working under fedora?

**Comment by santolucito** [comment]
I managed to get it working by installing SuperDirt, Dirt-Samples, and Vowels by cloning the repos into the Extensions directory and removing the dependencies of SuperDirt. That is, I changed this line https://github.com/musikinformatik/SuperDirt/blob/51eb2e220c42c708ea4432331b8503e54bf8cdfe/SuperDirt.quark#L6 to read

```	\dependencies:  [] ```

It seems 3.10.0 might have issues reading dependencies in certain situations? Very strange.
